ush/fv3gfs_downstream_nems.sh

```diff
@@ -158,193 +157,136 @@ while [ $nset -le $totalset ]; do
     # poescript for remaining processors
     if [[ $end -eq $ncount ]] ;then
       while [[ $iproc -lt $nproc ]];do
-        export iproc=`expr $iproc + 1`
-	echo "/bin/echo $iproc" >> $DATA/poescript
+        export iproc=$(expr $iproc + 1)
+        echo "/bin/echo $iproc" >> $DATA/poescript
       done
       break
     fi
-    export iproc=`expr $iproc + 1`
+    export iproc=$(expr $iproc + 1)
   done
 
-date
+  date
   chmod 775 $DATA/poescript
   export MP_PGMMODEL=mpmd
   export MP_CMDFILE=$DATA/poescript
   launcher=${APRUN_DWN:-"aprun -j 1 -n 24 -N 24 -d 1 cfp"}
   if [ $machine = WCOSS_C -o $machine = WCOSS_DELL_P3 -o $machine = WCOSS2 ] ; then
-     $launcher $MP_CMDFILE
+    $launcher $MP_CMDFILE
   elif [ $machine = HERA -o $machine = ORION -o $machine = JET -o $machine = S4 ] ; then
-     if [ -s $DATA/poescript_srun ]; then rm -f $DATA/poescript_srun; fi
-     touch $DATA/poescript_srun
-     nm=0
-     cat $DATA/poescript | while read line; do
-         echo "$nm  $line" >> $DATA/poescript_srun 
-         nm=$((nm+1))
-     done
-     ${launcher:-"srun --export=ALL"}  -n $nm --multi-prog $DATA/poescript_srun
+    if [ -s $DATA/poescript_srun ]; then rm -f $DATA/poescript_srun; fi
+    touch $DATA/poescript_srun
+    nm=0
+    cat $DATA/poescript | while read line; do
+      echo "$nm $line" >> $DATA/poescript_srun 
+      nm=$((nm+1))
+    done
+    ${launcher:-"srun --export=ALL"} -n $nm --multi-prog $DATA/poescript_srun
   else
-     $launcher
+    $launcher
   fi
   export err=$?
   if [ $err -ne 0 ]; then sh +x $DATA/poescript ; fi
 
-date
+  date
   export iproc=1
   while [ $iproc -le $nproc ]; do
     if [ $nset = 1 ]; then
-     cat pgb2file_${fhr3}_${iproc}_0p25 >> pgb2file_${fhr3}_0p25
-     if [ "$PGBS" = "YES" ]; then
-       cat pgb2file_${fhr3}_${iproc}_0p5  >> pgb2file_${fhr3}_0p5
-       cat pgb2file_${fhr3}_${iproc}_1p0  >> pgb2file_${fhr3}_1p0
-       if [ "$PGB1F" = 'YES' ]; then
-         cat pgbfile_${fhr3}_${iproc}_1p0   >> pgbfile_${fhr3}_1p0
-       fi
-     fi
+      cat pgb2file_${fhr3}_${iproc}_0p25 >> pgb2file_${fhr3}_0p25
+      if [ "$PGBS" = "YES" ]; then
+        # cat pgb2file_${fhr3}_${iproc}_0p5  >> pgb2file_${fhr3}_0p5
```

Comment:
@WalterKolczynski-NOAA This is the operational GFS key products. You might add a switch to turn off for developer experiment.

Reply:
I had restored them all in the other script, I just missed these. I'm putting them back in for the g-w PR.